pkg/controller/virtualmachineimport/virtualmachineimport_controller.go

```diff
@@ -141,10 +151,15 @@ type ReconcileVirtualMachineImport struct {
 func (r *ReconcileVirtualMachineImport) Reconcile(request reconcile.Request) (reconcile.Result, error) {
 	reqLogger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
 	reqLogger.Info("Reconciling VirtualMachineImport")
+	cfg, err := r.kvConfigProvider.GetConfig()
+	if err != nil {
+		return reconcile.Result{}, err
+	}
+	reqLogger.Info("Using config", "KubeVirt", cfg)
```

Comment:
`cfg` is not used anywhere else than in the log.

Reply:
It is here just to show what configuration is used. Placement policy validation implementation will come in another PR.

Reply:
As we know it works we can remove it in this PR and add it in another when we improve validation.

Reply:
:+1:

Reply:
Done